microsoft-edge/toc.yml

```diff
@@ -510,6 +512,8 @@
           items:
             - name: Visual Studio Code overview
               href: visual-studio-code/index.md
+            - name: Open source files in Visual Studio Code
```

Comment:
@codepo8 - speaking of experiments, we are trying out having two entries in the TOC.  The UX seems to support this pretty well now.